src/UserConsentListBuilder.php

```diff
@@ -65,10 +66,12 @@ public function buildHeader() {
    * {@inheritdoc}
    */
   public function buildRow(EntityInterface $entity) {
+    /** @var \Drupal\user\UserInterface $owner */
```

Comment:
This line isn't needed because the type of this variable is checking further in the code

Reply:
Fixed